data/json/items/gun/36paper.json

```diff
@@ -4,8 +4,16 @@
     "copy-from": "pistol_revolver_cap_ball",
     "looks_like": "sw_619",
     "type": "GUN",
-    "name": { "str_sp": "Colt M1861 Navy" },
+    "name": { "str_sp": "civil war six-shooter" },
     "description": "The Colt Model 1861 Navy cap & ball .36-caliber revolver was a six-shot, single-action percussion weapon produced by Colt's Manufacturing Company from 1861 until 1873.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "colt_navy",
+        "name": { "str": "Colt M1861 Navy", "str_pl": "Colt M1861 Navies" },
+        "description": "The Colt M1861 is a black powder, cap & ball revolver dating back to the Civil War.  While reloading it takes quite awhile due to having to put a primer on each chamber manually, instead of having the primer in one unit with the casing, like metallic cartridges.  However, for the time 6 shots in quick succession was a great thing for cavalry and officers to have on the field, though this version was much rarer than the more popular Colt M1860.  Nowadays, this firearm is completely outdated, and only sees use in collections, reenactment, or special shooting competitions.  However, black powder guns such as this are less regulated than more modern firearms, so it is sometimes the first gun a youth may buy.  This one is chambered in .36, weaker than the M1860's .44."
```

Comment:
```suggestion
        "description": "The Colt M1861 is a black powder, cap & ball revolver dating back to the Civil War.  While reloading it takes quite a while due to having to put a primer on each chamber manually, instead of having the primer in one unit with the casing, like metallic cartridges.  However, for the time 6 shots in quick succession was a great thing for cavalry and officers to have on the field, though this version was much rarer than the more popular Colt M1860.  Nowadays, this firearm is completely outdated, and only sees use in collections, reenactment, or special shooting competitions.  However, black powder guns such as this are less regulated than more modern firearms, so it is sometimes the first gun a youth may buy.  This one is chambered in .36, weaker than the M1860's .44."
```